src/utils/getMaxOffset.ts

```diff
@@ -0,0 +1,6 @@
+export const getMaxOffset = (zoom: number, borderExists: boolean = false) => {
+    const maxOffset: number = ((zoom || 100) / 100 - 1) * (borderExists ? 576 : 636)
```

Comment:
Good change.

We decided to only go with the 576 number as we'd have to duplicate this in the contract and it is not as straightforward there. 576 pixels of movement is plenty even is the border doesn't exist.

Reply:
576 is plenty, makes it cleaner to use on the frontend as well. Fixed